packages/cxl-ui/scss/cxl-login.scss

```diff
@@ -0,0 +1,10 @@
+.login #nav a,
+.login #backtoblog a {
+  color: #2271b1;
```

Comment:
Can you check what would be the size of a bundle that includes only cxl-lumo-styles/color.js?

Ideally we would be able to build a minimal bundle of at least color and typography for specialty screens like login.

Reply:
For now we don't need such advanced checks. We just need **red links** on login page. That's it. When redesign time come we'll think about system for it.

Reply:
> checks

Wdym "checks" here?

Reply:
I don't know honestly. I was tired.